pkg/helpers/imageregistry/client.go

```diff
@@ -56,13 +56,19 @@ func NewClient(kubeClient kubernetes.Interface) Interface {
 }
 
 func (c *Client) Cluster(cluster *clusterv1.ManagedCluster) Interface {
```

Comment:
Should we return the error?

Reply:
no error returned because want to make it easy use `client.Cluster(cluster).PullSecret(),` `client.Cluster(cluster).ImageOverride(image)`.
if there error occurred here, will return error in  PullSecret(), and return original image in ImageOverride(image)